speed-dreams/source-2.2.3/src/libs/tgfclient/gui.h

```diff
@@ -230,6 +230,32 @@ typedef struct
     tfuiCheckboxCallback onChange;
 } tGfuiCheckbox;
 
+// SIMULATED DRIVING ASSISTANCE CHANGE: Added radiobutton type and radiobutton-list type
+/// @brief Radio Button type
+typedef struct
+{
+    int labelId;                      /**< id of the text label                    */
+    int parentId;                     /**< id of the radiobutton-list parent       */
+    int listId;                       /**< local id of the parent radiobutton-list */
+    void* scr;                        /**< The screen the radiobutton is on        */
+    tRadioButtonInfo* pInfo;          /**< Radiobutton information                 */
+
+    GfuiColor fgColor[3];             /**< Color */
+    int selectedId;                   /**< id of the selected radiobutton   */
+    int notSelectedId;                /**< id of the unselected radiobutton */
+
+    tfuiRadioButtonCallback onChange; /**< Function to call when a radio button is clicked */
```

Comment:
Rename to fit naming standards. Possibly should also rename id's to controls
```suggestion
    int LabelId;                      /**< id of the text label                    */
    int ParentId;                     /**< id of the radiobutton-list parent       */
    int ListId;                       /**< local id of the parent radiobutton-list */
    void* Scr;                        /**< The screen the radiobutton is on        */
    tRadioButtonInfo* Info;          /**< Radiobutton information                 */

    GfuiColor FgColor[3];             /**< Color */
    int SelectedId;                   /**< id of the selected radiobutton   */
    int NotSelectedId;                /**< id of the unselected radiobutton */

    tfuiRadioButtonCallback OnChange; /**< Function to call when a radio button is clicked */
```